smart_fetch/cli/export.py

```diff
@@ -231,9 +232,15 @@ async def run_hydration_tasks():
                 if input_type not in loop_types:
                     continue
 
+                if first:
+                    rich.get_console().rule()
+                    first = False
```

Comment:
nit - why not just do this outside the while loop?

Reply:
Ah totally reasonable question - but part of the while loop is determining if the task should be run - so we only want to print this if we actually do run any of the tasks inside the loop.